test/internals/validation.test.js

```diff
@@ -82,6 +82,7 @@ test('build schema - payload schema', t => {
 test('build schema - avoid repeated normalize schema', t => {
   t.plan(3)
   const opts = {
+    jsonShorthand: true,
```

Comment:
why this has been added here?

I would move this in a dedicated `serverConfig` var